mqtt_test.py

```diff
@@ -59,8 +59,8 @@ def on_resubscribe_complete(resubscribe_future):
 
 receive_results = {}
 receive_event = threading.Event()
-def on_receive_message(**kwargs):
-    receive_results.update(kwargs)
+def on_receive_message(topic, payload, **kwargs):
+    receive_results.update(locals())
```

Comment:
I'd suggest reverting this one, since you're not doing anything with the named variables anyway...

Reply:
I made this more explicit and less weird